lib/process-services/task-list/components/task-filters.component.ts

```diff
@@ -186,27 +191,26 @@ export class TaskFiltersComponent implements OnInit, OnChanges {
 
     /**
      * Select the first filter of a list if present
+     * @param filterParam
+     * @param filteredFilterList
      */
     public selectTaskFilter(filterParam: FilterParamsModel, filteredFilterList: FilterRepresentationModel[]) {
```

Comment:
I'd like to have this method as private, in this way there is no way to call it from outside.

Reply:
Done.